README.md

```diff
@@ -21,8 +21,17 @@ composer require escopecz/mautic-form-submit
 ## Usage
 
 ```php
+//basic form data
+$email		= $_REQUEST['email'] ?? $_REQUEST['Email'] ?? $_REQUEST['E-mail'] ?? $_REQUEST['E-MAIL']; // mauticform_input_mkregistraciaslp_email
+$name		  = $_REQUEST['name'] ?? $_REQUEST['Name']; // mauticform_input_mkregistraciaslp_name
+$phone		= $_REQUEST['phone'] ?? $_REQUEST['Phone']; // mauticform_input_mkregistraciaslp_phone
+$form_id	= $_REQUEST['form_id'] ?? $_REQUEST['Form_id']; // mauticform_input_mkregistraciaslp_form_id
+if (is_null($form_id)) {$form_id  = 1;} // if form_id is empty then use the 1'st Mautic form
+$form   = $mautic->getForm($form_id);
```

Comment:
It must be a real Mautic form. The specific form that has the fields defined. It cannot default to 1.

Reply:
Again, to speed up the start of work if the transmitter is not on our side.  We can send a link to our script, create 1 form and immediately start accepting data.  And then when the form ids are configured on an external resource, the data will begin to be distributed to the required forms in mautic